src/deploy/campaign/get-config.ts

```diff
@@ -82,7 +92,7 @@ export const getConfig = async ({
 
   let zeroVaultAddressConf : string;
```

Comment:
small nitpick, but what is the `Conf` suffix adding here? Makes me think this is a config, but it's just a single value. I would just use `zeroVaultAddress`